contracts/StrategyRegistry.vy

```diff
@@ -0,0 +1,61 @@
+# @version 0.2.12
+
+governance: public(address)
+pendingGovernance: public(address)
+
+event NewGovernance:
+    governance: address
+
+interface Strategy:
+    def name() -> String[64]: view
+    def apiVersion() -> String[8]: view
+
+struct StrategyInfo:
+  name: String[64]
+  apiVersion: String[8]
+
+strategyVersions: public(HashMap[StrategyInfo, address])
```

Comment:
AFAIK, to query this mapping we will always need the name and apiVersion to get the strategy address. Right?

Reply:
Yes